locale/texts.pot

```diff
@@ -48,7 +48,7 @@ msgid "[Last] dddd [•] HH:mm"
 msgstr ""
 
 #: src/models.js:107
-#: src/utils.js:426
+#: src/utils.js:427
```

Comment:
Why this changed? You didn't change `src/utils`

Reply:
I ran an update, probably it was lacking an update.